File: Python/exer11.py
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|
#Emiliano Costa Junior - RA:1260901923008
#FATEC - Automação Industrial
#Programação Aplicada a Automação
#Programa 11 - Seleciona o maior valor digitado
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|
#Descrição: 
#Entrar com dois valores quaisquer. Exibir o maior deles, se existir, caso contrário, enviar mensagem avisando que os 
# números são idênticos.
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|

#Mensagem de Apresentação
print("Seleciona o menor valor digitado")

#Entrada dados
valor1 = float(input("Digite o valor1: "))
valor2 = float(input("Digite o valor2: "))

#escolhe o maior
if(valor1 == valor2):
    print(f"Os dois numeros tem valor de:{valor1: .2f}")
else:
    if(valor1 > valor2):
        print(f"maior valor é:{valor1: .2f}")
    else:
        print(f"maior valor é:{valor2: .2f}")

#Aguarda o usuario finalizar
fim = input("clique qualquer tecla para finalizar!")
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|

File: exer_11.c
/*******1|********2|********3|********4|********5|********6|********7|********8|
'########:'##::::'##:'####:'##:::::::'####::::'###::::'##::: ##::'#######::
 ##.....:: ###::'###:. ##:: ##:::::::. ##::::'## ##::: ###:: ##:'##.... ##:
 ##::::::: ####'####:: ##:: ##:::::::: ##:::'##:. ##:: ####: ##: ##:::: ##:
 ######::: ## ### ##:: ##:: ##:::::::: ##::'##:::. ##: ## ## ##: ##:::: ##:
 ##...:::: ##. #: ##:: ##:: ##:::::::: ##:: #########: ##. ####: ##:::: ##:
 ##::::::: ##:.:: ##:: ##:: ##:::::::: ##:: ##.... ##: ##:. ###: ##:::: ##:
 ########: ##:::: ##:'####: ########:'####: ##:::: ##: ##::. ##:. #######::
........::..:::::..::....::........::....::..:::::..::..::::..:::.......:::
Arquivo: exer_11.C
Autor: Emiliano Costa Junior
Data: 23 de Agosto 2021 
Descrição: Entrar com dois valores quaisquer. Exibir o maior deles, se existir, 
caso contrário, enviar mensagem avisando que os números são idênticos.
------------------------------------------------------------------------------*/
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(int argc, char const *argv[]){
//Inicia
  setlocale(LC_ALL, "Portuguese");
  printf("Exercicio 11:\t");

//Entrada de dados 
  printf("Exibe o menor\n");
  printf("Digite o primeiro valor: ");
  int d_valorA;
  scanf("%d",&d_valorA);
  printf("Digite o segundo valor: ");
  int d_valorB;
  scanf("%d",&d_valorB);

//decisão
	if(d_valorA==d_valorB)
	{
		goto iguais; //Testando o comando goto
	}
	else
	{
		if(d_valorA>d_valorB)
			goto maiorA;
		else
			goto maiorB;
	}

//Exibe resultado final
iguais:
	printf("Valores são iguais %d", d_valorA);
	goto fim;
maiorA:
	printf("Maior valor é %d", d_valorA);
	goto fim;
maiorB:
	printf("Maior valor é %d", d_valorB);
fim:
//Termina
  return (EXIT_SUCCESS);
}/*FIM MAIN*/
/*Fim de Arquivo-------------------------------------------------------------*/
